### DISCUSSION TITLE
Wiki 可能需要更新

### DISCUSSION THREAD

**Discussion by libook** [vernesong/OpenClash#2857] [General]
Wiki 的[常规设置](https://github.com/vernesong/OpenClash/wiki/%E5%B8%B8%E8%A7%84%E8%AE%BE%E7%BD%AE)部分描述了OpenClash 运行模式分为：
- 增强/兼容
- TUN
- 游戏

而当前实际上界面上显示的模式为：
- 增强
- TUN
- 混合

作为新手不了解这块的信息是如何对应的，不知道Wiki的内容是否需要更新？

**Comment by alanhe421** [comment]
我这里显示的是

1. Redir-.Host(兼容)模式
2. Redir-Host(TUN)模式
3. Redir-Host(TUN-混合)模式【UDP-TUN,TCP-转发】

![image](https://user-images.githubusercontent.com/9245110/210164026-69d48b5f-3311-43ee-909f-7da29fc7b742.png)


## version info
Current Version： v0.45.78-beta
